cloudflare/schema_cloudflare_fallback_domain.go

```diff
@@ -10,14 +10,29 @@ func resourceCloudflareFallbackDomainSchema() map[string]*schema.Schema {
 			Type:     schema.TypeString,
 			Required: true,
 		},
+		"include_default_domains": {
+			Optional: true,
+			Type:     schema.TypeBool,
+			Default:  true,
+		},
+		"restore_default_domains_on_delete": {
```

Comment:
i'm not sure if this attribute makes sense to exist. from the service perspective, the resource is always present we are just modifying it (even before it is configured by the end user). why would we not just update the `Delete` method to send only the default domains instead?